code/modules/exploration_crew/discovery_research/discovery_scanner.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A scanner used by scientists to collect research data about unknown artifacts and specimins."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "discovery"
-	item_state = "healthanalyzer"
+	item_state = "discoveryscanner"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
```

Comment:
I feel like it should be moved out of `medical_`, but I'm not sure what the convention is.

Reply:
when i checked i didn't see a sci one. i could put it in mining but meh